sklearn/metrics/scorer.py

```diff
@@ -225,18 +225,13 @@ def get_scorer(scoring):
     scorer : callable
         The scorer.
     """
-    valid = True
```

Comment:
Perhaps we should leave this code in in case of future deprecations. Or else just note somewhere that if deprecations are needed, SCORER can be redefined as a DeprecationDict, but `keys` etc then need to be reimplemented to hide deprecated keys.